prebuild.py

```diff
@@ -112,16 +115,39 @@ def main():
             del os.environ[var]
 
     args = parse_args()
+    assets_url = hifi_utils.readEnviromentVariableFromFile(args.build_root, 'EXTERNAL_BUILD_ASSETS')
 
     if args.ci_build:
         logging.basicConfig(datefmt='%H:%M:%S', format='%(asctime)s %(guid)s %(message)s', level=logging.INFO)
 
     logger.info('sha=%s' % headSha())
     logger.info('start')
 
+    # OS dependent information
+    system = platform.system()
+    if 'Windows' == system and 'CI_BUILD' in os.environ and os.environ["CI_BUILD"] == "Github":
+        logger.info("Downloading NSIS")
+        with timer('NSIS'):
+            hifi_utils.downloadAndExtract(assets_url + '/dependencies/NSIS-hifi-plugins-1.0.tgz', "C:/Program Files (x86)")
+
+    qtInstallPath = ''
+    # If not android, install our Qt build
+    if not args.android:
+        qt = hifi_qt.QtDownloader(args)
+        qtInstallPath = qt.cmakePath
+        with hifi_singleton.Singleton(qt.lockFile) as lock:
+            with timer('Qt'):
+                qt.installQt()
+                qt.writeConfig()
+
     # Only allow one instance of the program to run at a time
```

Comment:
Move this comment down to just above `with hifi_singleton…`.

Reply:
ah, good catch, not sure how it got here